dev-server.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  publicPath: 'http://localhost:8080/dist/',
```

Comment:
I think it makes sense to provide a decent default for the dev server, but
we could at least make the `http://localhost:8080` overwritable by an ENV variable I think. Sth like this?

```
`${process.env.DEV_HOST || 'http://localhost:8080'}/dist/`
```

(or whatever might be a better name for `DEV_HOST`)